S7.Net/AutoReconnector.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Timers;
```

Comment:
Not sure why you used `System.Timers.Timer`, but I think `System.Threading.Timer` is the better choice in this case.

Reply:
Not just a better choice, probably the only real one 😊